x-pack/plugins/fleet/public/applications/integrations/sections/epm/screens/detail/documentation/index.tsx

```diff
@@ -34,8 +35,38 @@ interface Props {
   integration?: string | null;
 }
 
+const getInputs = ({ packageInfo, integration }: Props) => {
```

Comment:
this is pulled out from the `Inputs` component below so that it can be used in `hasDocumentation`, which is used by the main details page to determine if the tab should be shown

Reply:
Could we cover the changes with unit tests?